apps/prairielearn/src/pages/instructorAssessmentQuestions/components/InstructorAssessmentQuestionsTable.tsx

```diff
@@ -47,81 +58,915 @@ function Title({
+      null;
+  }
+
+  return (
+    <Fragment>
+      <tr>
+        {editMode ? (
+          <>
+            <td class="align-content-center">
+              <button
+                class="btn btn-sm btn-ghost"
+                type="button"
+                onClick={() => {
+                  handleEditQuestion({
+                    question,
+                    alternativeGroup: alternative ? alternativeGroup : undefined,
+                    zoneNumber,
+                    alternativeGroupNumber,
+                    alternativeNumber,
+                  });
+                }}
+              >
+                <i class="fa fa-edit" aria-hidden="true" />
+              </button>
+            </td>
+            <td class="align-content-center">
+              <button
+                class="btn btn-sm btn-ghost"
+                type="button"
+                onClick={() =>
+                  handleDeleteQuestion(
+                    zoneNumber,
+                    alternativeGroupNumber,
+                    questionId,
+                    alternativeNumber,
+                  )
+                }
+              >
+                <i class="fa fa-trash text-danger" aria-hidden="true" />
+              </button>
+            </td>
+          </>
+        ) : (
+          ''
+        )}
+        <td>
+          <Title
+            questionRow={questionData}
+            hasCoursePermissionPreview={hasCoursePermissionPreview ?? false}
+            urlPrefix={urlPrefix}
+            questionNumber={questionNumber}
+            alternativeNumber={displayAlternativeNumber}
+          />
+          <IssueBadge
+            urlPrefix={urlPrefix}
+            count={questionData.open_issue_count}
+            issueQid={questionData.question.qid}
+          />
+        </td>
+        <td>
+          {questionData.question.sync_errors ? (
+            <SyncProblemButton output={questionData.question.sync_errors} type="error" />
+          ) : questionData.question.sync_warnings ? (
+            <SyncProblemButton output={questionData.question.sync_warnings} type="warning" />
+          ) : (
+            ''
+          )}
+          {questionId}
+        </td>
+        <td>
+          <TopicBadge topic={questionData.topic} />
+        </td>
+        <td>
+          <TagBadgeList tags={questionData.tags} />
+        </td>
+        <td>
+          {maxPointsText({
+            max_auto_points: maxAutoPoints ?? null,
+            max_manual_points: question.manualPoints ?? null,
+            auto_points:
+              question.points ?? question.autoPoints ?? alternativeGroupAutoPoints ?? null,
+            assessmentType,
+          })}
+        </td>
+        <td>{question.manualPoints || '—'}</td>
+        {showAdvanceScorePercCol ? (
+          <td
+            class={clsx(
+              questionData.assessment_question.effective_advance_score_perc === 0
+                ? 'text-muted'
+                : '',
+            )}
```

Comment:
Nit, this is a slightly more conventional way to write this with `clsx`:

```suggestion
            class={clsx({
              'text-muted': questionData.assessment_question.effective_advance_score_perc === 0,
            })}
```

If you want the original way, you don't need `clsx` at all.

Reply:
Updated to this locally in [9c6a8dd](https://github.com/PrairieLearn/PrairieLearn/pull/13171/commits/9c6a8dd7990dc375ae5e1d6b567c3c4c46b50e22).